back-end/package.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
We don't want `package.json`  and `package-lock.json` in backend folder unless absolutely necessary. Why do we need lerna here?